themes/finna2/js/finna-layout.js

```diff
@@ -373,11 +373,22 @@ finna.layout = (function finnaLayout() {
 
     holder.find('.condensed-collapse-toggle').off('click').click(function onClickCollapseToggle(event) {
       if ((event.target.nodeName) !== 'A' && (event.target.nodeName) !== 'MARK') {
-        $(this).nextAll('.condensed-collapse-data').first().slideToggle(120, 'linear');
-        $('.fa-arrow-right', this).toggleClass('fa-arrow-down');
         holder = $(this).parent().parent();
         holder.toggleClass('open');
-        if (holder.hasClass('open') && !holder.hasClass('opened')) {
+
+        var onSlideComplete = function onSlideComplete () {
+          if (holder.hasClass('open') && holder.hasClass('opened')) {
```

Comment:
Toi opened-tarkistus pitäisi tehdä ennen slideToggle-kutsua, varmuuden vuoksi. Ja toisin päin? Oikeastaan koko onSlideCompleten voi jättää tyhjäksi tai pois slideToggle-kutsusta, jos holder:lla on jo opened-luokka.

Reply:
Poistin tarkistukset callbackista. Callbackia kutsutaan nyt vain avattessa.